src/network/interfaces_default.go

```diff
@@ -1,4 +1,5 @@
 //go:build !linux
+// +build !linux
```

Comment:
?

Reply:
This was needed for it to compile. Which version of go did you complie it with?

Reply:
1.17.6 and goland is not complaining about that.. 🤔

Reply:
If there is a reason I am fine with that, but so far it was not needed

Reply:
My Goland and version 1.16 does

Reply:
yes, it's something solved in go 1.17, I will update my version.